principles/work_in_the_open.md

```diff
@@ -0,0 +1,49 @@
+
+> You must make all new source code open to meet point 8 of the Digital Service Standard.
+
+So you could develop your code in private, and release in time for a [Beta](https://www.gov.uk/service-manual/agile-delivery/how-the-beta-phase-works) assessment, which (other factors aside) would be sufficient to pass.
+
+However we believe there are more benefits to making code public from the beginning, rather than starting private and releasing later.
+
+## Benefits
+
+### Free stuff
+
+Public repositories benefit from lots of free services. Continuous integration, dependency checking, code quality and test coverage are just a few of the things you can get for free if your project is open.
+
+### You've passed
+
+...point 8 of the Digital service standard (if you're a digital service going on [GOV.UK](https://gov.uk)). No last minute rush to release something just before an assessment, you're sorted from day one.
+
+### Flexibility
+
+It's easier to collaborate with colleagues when working on an open repo compared to a private one. Though you may have to grant access to allow them to make changes, there is nothing stopping them from looking at and referring to the code.
+
+### Visibility and sharing
+
+Projects that are open have more visibility which makes it easier for colleagues both In the Environment Agency and in other departments to see what we've built. This encourages collaboration rather than duplication.
+
+It also means the code can benefit more people by being shared. Not only can others use your code they can see which problems you have already solved.
+
+### No re-work
+
+In our experience all projects that started private with the intention of going public later found that additional work was needed before this could happen. Projects that started public however never seem to experience this overhead.
+
+Starting public prevents you from storing up additional work which has to be done, normally at a time where you're priority is actually to get it live.
+
+### Quality
+
+We believe the quality of public projects is better than private ones. Our (unscientific!) opinion is based on the belief that
+
+- Access to quality metrics means contributors are less likely to commit code without test coverage
+- Documentation is created and updated from the start rather than later, for example when going from private to public
+- Developers make more balanced decisions between delivery and quality due to the possibility of public scrutiny
+
+## Authority to 'go open'
+
+When writing code on a project, authority to release that code falls under the data custodian for the area of the business the service belongs to. Scheduling a meeting with the data custodian should be a priority in sprint 1, so you can get your code open and take advantage of the benefits above as soon as possible.
```

Comment:
In practice, this could be a process that might take a little while.  We might want some slightly different guidance for Alpha and Beta - at the start of Alpha you should seek to get this approval, whereas at the start of Beta you should already have it (and in fact it should be a precondition of starting Beta)?